mnefun/_mnefun.py

```diff
@@ -1122,6 +1110,33 @@ def run_sss_locally(p, subjects, run_indices):
             raw_sss.save(o, overwrite=True, buffer_size_sec=None)
 
 
+def _load_trans_to(p, subj, run_indices, raw=None):
+    if isinstance(p.trans_to, string_types):
+        if p.trans_to == 'median':
+            trans_to = op.join(p.work_dir, subj, p.raw_dir,
+                               subj + '_median_pos.fif')
+            if not op.isfile(trans_to):
+                calc_median_hp(p, subj, trans_to, run_indices)
+        trans_to = mne.read_trans(trans_to)
```

Comment:
Currently fails in case where `trans_to == 'twa'`

Reply:
it should, #200 is not in `master`

Reply:
I figured it will once that is merged. Just making sure.